docs/pages/access-controls/access-request-plugins/ssh-approval-opsgenie.mdx

```diff
@@ -0,0 +1,114 @@
+  the 'APP settings' section and selecting API key management.
+
+- This feature is currently only available for Cloud isntances.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create services
+
+To demonstrate the Opsgenie plugin, create an Opsgenie team named
+`teleport-access-request-notifications`.
+
+We will configure the Opsgenie plugin to create an alert for the
+`teleport-access-request-notifications` team when certain users
+create an Access Request.
+
+## Step 2/5. Define RBAC resources
+
+The Teleport Opsgenie plugin works by receiving Access Request events from the
+Teleport Auth Service and, based on these events, interacting with the Opsgenie
+API.
+
+### Create a user who will request access
+
+Create a user called `myuser` who has the `editor-requester` role. Later in this
+guide, you will create an Access Request as this user to test the Opsgenie
+plugin:
+
+```code
+$ tctl users add myuser --roles=editor-requester
+```
+
+`tctl` will print an invitation URL to your terminal. Visit the URL and log in
+as `myuser` for the first time, registering credentials as configured for your
+Teleport cluster.
+
+## Step 3/5. Set up a Opsgenie API key
+
+Generate an API key that the Opsgenie plugin will use to create and modify
+alerts as well as list users, services, and on-call policies.
+
+In your Opsgenie dashboard, go to **SETTINGS → API SETTINGS** and click
+**API key management**. Enable 'read' and 'create and update'.
+
+![Create API key](../../../img/enterprise/plugins/opsgenie/opsgenie-api-key.png)
+
+## Step 4/5. Configure the Opsgenie plugin
+
+At this point, you have generated credentials that the Opsgenie plugin will use
+to connect to the Opsgenie API.
+
+![Integrations page](../../../img/enterprise/plugins/opsgenie/opsgenie-integration-page.png)
+
+## Step 5/5. Test the Opsgenie plugin
+
+### Create an Access Request
+
+As the Teleport user `myuser`, create an Access Request for the `editor` role:
+
+(!docs/pages/includes/plugins/create-request.mdx!)
+
+In Opsgenie, you will see a new alert containing information about the
+Access Request:
+
+### Resolve the request
+
+(!docs/pages/includes/plugins/resolve-request.mdx!)
+
+<Admonition title="Auditing Access Requests">
+
+When the Opsgenie plugin sends a notification, anyone who receives the
+notification can follow the enclosed link to an Access Request URL. While users
+must be authorized via their Teleport roles to review Access Request, you
+should still check the Teleport audit log to ensure that the right users are
+reviewing the right requests.
+
+When auditing Access Request reviews, check for events with the type `Access
+Request Reviewed` in the Teleport Web UI <ScopedBlock scope={[
+"enterprise"]}>and `access_request.review` if reviewing the audit log on the
+Auth Service host</ScopedBlock>.
+
+</Admonition>
+
+## Feedback
+
+If you have any issues with this plugin please create a [GitHub
+issue](https://github.com/gravitational/teleport-plugins/issues/new).
```

Comment:
Nit: Don't know if this is really necessary. None of Opsgenie stuff lives in teleport-plugins and besides I'm hoping this repo will go away entirely in future.
```suggestion
```